src/AssetUpload.js

```diff
@@ -35,10 +35,9 @@ class AssetUpload extends Component{
 
   render() {
 
-    let item2 = <Item message="a new message" />;
-    let item3 = <Item message="another message" />;
-    let item4 = <Item message="one more task" />;
-    let allTheThings = [item2, item3, item4];
+    let item2 = <Item message="JackpotJoy" />;
```

Comment:
would recommend using more of `const` instead of 'let`

Reply:
addressed